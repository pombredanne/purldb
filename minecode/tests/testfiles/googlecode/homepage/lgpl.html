

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "assetHostPath": "http://www.gstatic.com/codesite/ph", "loggedInUserEmail": null, "relativeBaseUrl": "", "domainName": null, "profileUrl": null, "projectName": "daumopeneditor", "token": null, "projectHomeUrl": "/p/daumopeneditor"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-36515974-1'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 daumopeneditor -
 
 
 semantic web editor - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fdaumopeneditor%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fdaumopeneditor%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/daumopeneditor">
 <a href="/p/daumopeneditor/">
 
 
 <img src="/p/daumopeneditor/logo?cct=1396244688"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/daumopeneditor/"><span itemprop="name">daumopeneditor</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/daumopeneditor/"><span itemprop="description">semantic web editor</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/daumopeneditor/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/daumopeneditor/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/daumopeneditor/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/daumopeneditor/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/daumopeneditor/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/daumopeneditor/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/daumopeneditor/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/daumopeneditor/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.gnu.org/licenses/lgpl.html" rel="nofollow">GNU Lesser GPL</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:daum">daum</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:webeditor">webeditor</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:semantic">semantic</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:Microformat">Microformat</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:RDF">RDF</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:openapi">openapi</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/101265759635984032355/">maart...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/101350114074915277517/">azk...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/102721630660490473512/">ragir...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/114291373914938849545/">seongak...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/107586401403363263555/">east...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/104723610382279476424/">tadoli2...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/117056275181314833357/">hmy3...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/108919073072673973649/">cha...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/108368587082930674131/">a...@azki.org</a
>, 
 
 

 <a class="userlink"
 href="/u/daumftdev/">daumftdev</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">11 committers</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="http://code.google.com/p/open-source-class" rel="nofollow">제주대 오픈소스 개발방법론</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>Groups</b></li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/daumopeneditor" rel="nofollow">메일링리스트</a>
 </li>
 
 <li class="psgap"/>
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <h1><a name="Daum_Open_Editor"></a>Daum Open Editor<a href="#Daum_Open_Editor" class="section_anchor"></a></h1><h1><a name="We&#x27;re_moving_to_Github!"></a>We&#x27;re moving to Github!<a href="#We&#x27;re_moving_to_Github!" class="section_anchor"></a></h1><p><a href="https://github.com/daumcorp/DaumEditor" rel="nofollow">https://github.com/daumcorp/DaumEditor</a> </p><p><table class="wikitable"><tr><td style="border: 1px solid #ccc; padding: 5px;"> <img src="http://code.google.com/images/code_logo.gif" /> </td><td style="border: 1px solid #ccc; padding: 5px;"> <img src="http://4.bp.blogspot.com/-GsgVU6l1Ixg/Tl-yRCtqJ4I/AAAAAAAACVc/EEAUaHaaMu4/s1600/movingtruck2.jpg" /> </td></tr> </table></p><p>Daum Editor가 <a href="https://github.com/daumcorp/DaumEditor" rel="nofollow">GitHub</a> 로 옮겨갑니다.  구글코드에 있는 다음에디터는 더 이상 유지되고 있지 않습니다. 문의는 <a href="https://github.com/daumcorp/DaumEditor" rel="nofollow">GitHub</a>에서 해주세요. </p><h2><a name="Introduction"></a>Introduction<a href="#Introduction" class="section_anchor"></a></h2><p><strong> Daum Open Editor </strong> 는 <a href="http://www.daum.net" rel="nofollow">Daum</a> 에서 개발된 WYSIWYG기반의 오픈 웹 에디터로 독립적이고 확장에 유연하도록 설계 되었습니다.<br>  주요 브라우저의 크로스브라우징을 지원하며, Daum에서 서비스되고 있는 Daum Editor와 함께 업데이트 될 예정입니다.<br><br> 다음오픈에디터의 라이센스는 GNU LGPL(Lesser General Public License) 으로 오픈되어 있는 소스이므로<br> 저작권료 없이 사용이 가능하며, 목적에 맞게 수정하여 사용할 수 있으십니다.<br> 또한 LGPL에는 수정한 부분의 소스공개를 권장하고 있으나, 강제 사항은 아니므로 공개하지 않으셔도 무방합니다. <br> 다만 사용하시는 소스 상단 부분에 다음오픈에디터를 사용하였음을 명시해 주시길 권장합니다.<br><br> <img src="http://uie.daum.net/openeditor/wiki/web_browser.PNG" /><br> IE 5.5+ 지원 (safari 3.2 + , chrome 1, ff 2+, opera 9.64+) </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

