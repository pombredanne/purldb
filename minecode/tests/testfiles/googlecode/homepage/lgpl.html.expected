{'content_license': None,
 'content_license_url': None,
 'description': u'semantic web editor',
 'downloads_url': u'http://code.google.com/p/daumopeneditor/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/daumopeneditor/feeds',
 'home_url': u'http://code.google.com/p/daumopeneditor/',
 'id': u'daumopeneditor',
 'issues_url': u'http://code.google.com/p/daumopeneditor/issues/list',
 'keywords': [],
 'labels': [u'Microformat',
            u'RDF',
            u'daum',
            u'openapi',
            u'semantic',
            u'webeditor'],
 'license': u'GNU Lesser GPL',
 'license_url': u'http://www.gnu.org/licenses/lgpl.html',
 'name': u'daumopeneditor',
 'peoples_url': u'http://code.google.com/p/daumopeneditor/people/list',
 'vcs_url': u'http://code.google.com/p/daumopeneditor/source/checkout',
 'wiki_url': u'http://code.google.com/p/daumopeneditor/w/list'}